### ISSUE TITLE
Strengths using value from webUI settings, not the 'Hypernetwork strengths' input box.

### ISSUE THREAD

**Issue by jwoeifjofwefawsfasd** [antis0007/sd-webui-multiple-hypernetworks#15]
As can be seen on the two screenshots, changing the value in the "hypernetwork Strengths" input box doesn't actually change the strengths, and thus doesn't change the image at all.

![image](https://user-images.githubusercontent.com/20150906/212221758-b48bc366-da45-44d8-a58b-66ffcaa9aece.png)
![image](https://user-images.githubusercontent.com/20150906/212221976-22a24224-ef40-492f-ace5-808a06774d92.png)

however, if I change the "hypernetwork strength" slider:
![image](https://user-images.githubusercontent.com/20150906/212222143-b511cb91-8c22-4a66-99cd-2ace47f4e1be.png)

then I see that it properly changes the image - note the dogs looking different compared to the previous two images.
![image](https://user-images.githubusercontent.com/20150906/212222281-359ec714-7a80-4606-a314-8c929fbc461f.png)

**TLDR**: the hypernetworks are using the strength value from the webui global settings, not the value set by this extension.

Thanks, and with love.

**Comment by jwoeifjofwefawsfasd** [comment]
Just tried using the script instead, seems like it works as intended - the images properly reflect the strengths inputted in the script's input box, and ignores the WebUI's "Hypernetwork Strength" setting.

**Comment by jwoeifjofwefawsfasd** [comment]
seems like this issue has already been brought up in the past and [has been previously fixed](https://github.com/antis0007/sd-webui-multiple-hypernetworks/issues/10). not sure why I'm seeing this problem on my side with the latest commit. Will investigate for a bit.

**Comment by jwoeifjofwefawsfasd** [comment]
Installing the [monkey patch extension](https://github.com/aria1th/Hypernetwork-MonkeyPatch-Extension) fixes this issue and the strengths are applied properly. 

Is this intended behavior? - by that, I mean is MonkeyPatch-Extension intended to be a dependency for this project?
